### PR TITLE
Add parameter to replace or hide form title; refs #924

### PR DESCRIPTION
**Pull request by trasher**

### CODE REVIEW

File: inc/commondbtm.class.php [hunk 1]
```diff
@@ -2279,89 +2280,91 @@ function showFormHeader($options=array()) {
       echo "<div class='spaced' id='tabsbody'>";
       echo "<table class='tab_cadre_fixe' id='mainformtable'>";
 
-      echo "<tr class='headerRow'><th colspan='".$params['colspan']."'>";
+      if ($params['formtitle'] !== '' && $params['formtitle'] !== false) {
```
Comment (orthagh) [inline inc/commondbtm.class.php]: is the check on empty string mandatory ?

Comment (trasher) [inline inc/commondbtm.class.php]: I've done that according to comments/specs defined on ticket #924 :)

Comment (orthagh) [inline inc/commondbtm.class.php]: ok, i missed that

**Comment by ddurieux** [comment]
So like I see, it's a little more than formtitle, it's formheader because you can disable recursive / entity information.

Perhaps modify it to have:

* name _noFormHeader_:
   * by default it's an empty array
   * and you can have these values:
       * _title_: used to rewrite title or if False disable display title
       * _entity_: true/false to display entities information (entity name, recursive)

**Comment by trasher** [comment]
A thing named "noSthing" with some values inside? I disagree ;p "formheader" in that case.
I guess it should also be set to "false" to hide entire header, the default value could stay null, I do not thing that change anything.

The question is; is it really usefull to hide only title or only entity informations? I do not want to add complexity without a reason.

**Comment by ddurieux** [comment]
I don't know, but seems more logical :p
So perhaps yes it's not a really good idea

**Pull request merged**